x-pack/plugins/fleet/common/constants/routes.ts

```diff
@@ -126,11 +128,15 @@ export const AGENTS_SETUP_API_ROUTES = {
   CREATE_PATTERN: `${API_ROOT}/agents/setup`,
 };
 
+export const RESET_API_ROUTES = {};
```

Comment:
Should the routes below in `PRECONFIGURATION_API_ROUTES` be here instead?

Reply:
Oops I forget to remove that useless variable.